runtime/fastly/builtins/fetch/fetch.cpp

```diff
@@ -11,6 +11,8 @@ using fastly::backend::Backend;
 using fastly::fastly::Fastly;
 using fastly::fetch::Request;
 
+using fastly::fastly::FastlyGetErrorMessage;
```

Comment:
Why did this namespace end up as `fastly::fastly`?

Reply:
This is basically `fastly::core` since each builtin has its own name under the `fastly::` namespace. We could do a rename to `fastly::core` in a follow-up.